docs/api-weightages/index.md

```diff
@@ -9,11 +9,15 @@ The number of requests you could make every month is appropriately limited in al
 Path &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; | Weightage
 --- | ---
 /v1/:chain/address/:address/assets | 20
+/v1/batch-assets | 1000
 /v2/:chain/address/:address/userData | 50
 /v1/:chain/address/:address/transactions | 5
-/v3/:chain/address/:address/transactions | 5
 /v2/:chain/address/:address/transactions | 20
+/v3/:chain/address/:address/transactions | 5
+/v1/:chain/address/:address/transactions/count | 50
```

Comment:
"/v1/:chain/address/:address/transactions/count": {
    "Weightage": 10,
    "tag": "core"
  },

Reply:
These have been updated in api service